### DISCUSSION TITLE
절대경로 변경 오류 (해결?)

### DISCUSSION THREAD

**Discussion by yeondooo** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#40] [Trouble Shooting]
https://velog.io/@welloff_jj/Applying-Absolute-Path-with-Craco-to-React-Project-made-with-CRA
이 링크 참고해서 했는데 된거 같은데 마지막에 craco.config.js파일 생성하는데 오류가 생기긴 합니다..
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/97602748/200780286-c2005dce-de83-480c-81fa-f665927b2d2a.png">

이건 에러메세지 보고 해결중이긴 한데, 절대경로로 불러오는건 됩니다!
<img width="489" alt="image" src="https://user-images.githubusercontent.com/97602748/200780366-61e1303f-e1f5-4228-bfbe-5e94cf841d96.png">

**Comment by baechanghyeon** [comment]
네네 저도 그 부분에서 에러가 나서 그냥 싹 다 밀었습니다... 연주님도 도전하셨군요..ㅋㅋ 
아마 맘 편히 다음 세션과제 때 다시 boilerPlate 업데이트 해서 활용해야 할거 같습니다 감사합니다 (__)(0 0)

**Reply by yeondooo** [reply]
require -> import 로 바꾸니 오류가 사라지긴 합니다.. 이렇게 써도 되는걸까요..?

**Reply by yeondooo** [reply]
craco.config.js
```
const CracoAlias = import('craco-alias');

module.exports = {
  plugins: [
    {
      plugin: CracoAlias,
      options: {
        source: 'tsconfig',
        tsConfigPath: 'tsconfig.paths.json',
      },
    },
  ],
};
```

**Reply by baechanghyeon** [reply]
엇 저도 Fix 누르니까 import로 변경이 되긴 하더라구요
혹시 import 부분이랑 안에 'craco-alias' 부분은 에러 안뜨시나요? 전 그쪽에 에러가 나서 그냥 접었는데 ㅎㅎ

**Reply by yeondooo** [reply]
그부분은 에러가 없는데 밑에 module.exports 부분에 이런 문구가 있긴 합니다..
<img width="728" alt="image" src="https://user-images.githubusercontent.com/97602748/200786343-b164d343-2f6a-4a85-a23f-0d000c6ffb5b.png">

**Reply by baechanghyeon** [reply]
ES 모듈로 바꿀 수 있다는건 선택사항인거 같아 무시해도 될거 같긴 합니다 !!
혹시 이거랑 json 파일 쪽에 script도 craco start 로 설정 해주셔야하는데 혹시 설정하시고 실행 해보셨나요????
저 그쪽에서 뭐 안되길래 때려치긴했습니다 ㅋㅋㅋㅋ

**Reply by yeondooo** [reply]
다 문제없이 됐는데 껐다 키니까 안됩니다.. 이제 용기를 내어 포기하겠습니다^_ㅠ

**Reply by baechanghyeon** [reply]
ㅋㅋㅋㅋㅋㅋㅋ수고하셨습니다...!!

**Comment by jaeyeong815** [comment]
연주님도 끈기 있으시군여!!! 창현님과 연주님께 한 수 배워갑니다 총총..
